### ISSUE TITLE
Keep Frei0r effects support for Olive RC1 release

### ISSUE THREAD

**Issue by Symbian9** [olive-editor/olive#767]
Please, keep Frei0r effects support for *Olive* `RC1` release (that should be released in next two month, before first stable release)

- https://github.com/olive-editor/olive/wiki/Frei0r-Support-EOL

Then drop it for next releases (`RC2`, `1.0`, and later)

**Comment by BrimsonBhin** [comment]
Olive is [**far** from a stable release](https://github.com/olive-editor/olive/projects). I believe a genuine release candidate might not happen into the early 2020s.

The `FurtherOCIO` included large changes that will effect Olive's interaction with Frei0r. Therefore, the library would just be an anchor to future development.

The costs outweigh the benefits in maintaining Frei0r for years only to get put down because some people want effects that could otherwise be ported to GLSL or something.

**Comment by alcomposer** [comment]
Which Frie0r effects do you work with regularly?

**Comment by Symbian9** [comment]
> I believe a genuine release candidate might not happen into the early 2020s.

I mean, lets create snapshot release (Alpha, Beta, etc.) before Frei0r support would be fully removed from Olive and keep it for historic reason with binary builds on GitHub:

- https://github.com/olive-editor/olive/releases

**Comment by unfa** [comment]
@Symbian9 What do you need the Frei0r effects for? Maybe thre's a better solution to your problem than keeping an old build of Olive around?

**Comment by Symbian9** [comment]
> Maybe there's a better solution to your problem than keeping an old build of Olive around?

Keeping old releases & builds is normal way of software development.

For example, *Blender* keep all builds (since 2003). So if user can't use Blender 2.8, then it's possible download older versions, that could be launched on older hardware and/or under older OS:

- https://download.blender.org/release/

So, I just want that there should be ***alpha release*** of *Olive* with `frei0r` support, before it would be fully removed from `master` branch.

**Comment by DaniSeeh** [comment]
I mean, it would be pretty easy to build old versions of Olive on Linux, for example. Every single commit is saved by GitHub. Are you wanting like, .exe or .appimage builds to be saved?

**Comment by Symbian9** [comment]
> I mean, it would be pretty easy to build old versions of Olive on Linux

Lets take in mind that most of users of Olive are content creators, not programmers

> Are you wanting like, .exe or .appimage builds to be saved?

Yes, I mean that there should be binary builds for users.

**Comment by DaniSeeh** [comment]
I mean, I know nothing about programming but it wasn't hard to follow the directions on the website and build Olive. The only extra step is downloading commit you want before building it. I don't see the harm in keeping old builds, but we probably don't want to encourage people to use them because they will consistently complain of already fixed bugs in newer builds. Olive really hasn't had any truly "stable" releases yet, it's very much in alpha and still deciding what it wants to be at that.

**Comment by oc1024** [comment]
I don't see how this would be better than support for software OpenGL.

**Comment by Symbian9** [comment]
> this would be better than support for software OpenGL.

I not say that this would be better.

I just tell that there should be pre-release (with binary byilds) with Frei0r support in "Releases" section on GitHub.

Then Frei0r could be fully deleted from `master` branch.

**Comment by itsmattkc** [comment]
I'll probably make a tagged release for the current master before merging in `furtherocio`. Mostly because the changes are already sweeping enough that current projects may not be 100% compatible with the newer system (I'll try to ensure they're as compatible as possible, but for safety it would be best to keep an old build accessible). I believe that will address this.
